src/main/js/helpers/constants.js

```diff
@@ -12,35 +12,41 @@ import * as DEFAULT_SHAPES from "../core/Shape/shapeDefinitions";
  *
  * @public
  * @property {string} BLACK "#1c1f21"
- * @property {string} BLUE "#007cc3"
- * @property {string} GREEN "#78c346"
- * @property {string} LIGHT_PURPLE "#c985da"
- * @property {string} ORANGE "#ffc20a"
+ * @property {string} BLUE "#176ba0"
+ * @property {string} GREEN "#076e00"
+ * @property {string} LIGHT_PURPLE "#bc70c7"
+ * @property {string} ORANGE "#b54900"
  * @property {string} YELLOW "#ffc20a"
+ * @property {string} PINK "#ae0868"
  * @property {string} GREY "#a7aaab"
- * @property {string} LIGHT_BLUE "#26a2e5"
- * @property {string} LIGHT_GREEN "#a5d784"
- * @property {string} PURPLE "#78288c"
- * @property {string} LIGHT_ORANGE "#ff7d00"
+ * @property {string} LIGHT_BLUE "#008fe0"
+ * @property {string} LIGHT_GREEN "#4b9b1c"
+ * @property {string} PURPLE "#9c19A3"
+ * @property {string} LIGHT_ORANGE "#e46511"
  * @property {string} LIGHT_YELLOW "#ffda6c"
+ * @property {string} LIGHT_PINK "#e16092"
  * @property {string} WHITE "#ffffff"
  * @enum {{Object: string}}
  */
+
 export const COLORS = {
     BLACK: "#1c1f21",
-    BLUE: "#007cc3",
-    GREEN: "#78c346",
-    LIGHT_PURPLE: "#c985da",
-    ORANGE: "#ed5e00",
+    BLUE: "#176ba0",
+    GREEN: "#076e00",
+    LIGHT_PURPLE: "#bc70c7",
+    ORANGE: "#b54900",
     YELLOW: "#ffc20a",
+    PINK: "#ae0868",
     GREY: "#a7aaab",
-    LIGHT_BLUE: "#26a2e5",
-    LIGHT_GREEN: "#a5d784",
-    PURPLE: "#78288c",
-    LIGHT_ORANGE: "#ff7d00",
+    LIGHT_BLUE: "#008fe0",
+    LIGHT_GREEN: "#4b9b1c",
+    PURPLE: "#9c19A3",
+    LIGHT_ORANGE: "#e46511",
     LIGHT_YELLOW: "#ffda6c",
+    LIGHT_PINK: "#e16092",
     WHITE: "#ffffff"
```

Comment:
nit: alphabetical order of colors

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/214/files#diff-965ff141b3e1c375076dd23d7144dd8aR14)